src/BenchmarksApps/TLS/Kestrel/Program.cs

```diff
@@ -157,4 +192,35 @@ static IPEndPoint CreateIPEndPoint(UrlPrefix urlPrefix)
     }
 
     return new IPEndPoint(ip, urlPrefix.PortValue);
+}
+
+static SslProtocols? ParseSslProtocols(string? supportedTlsVersions)
+{
+    var protocols = SslProtocols.None;
+    if (string.IsNullOrEmpty(supportedTlsVersions) || supportedTlsVersions == "any")
+    {
+        return null;
+    }
+
+    foreach (var version in supportedTlsVersions.Split(','))
+    {
+        switch (version.Trim().ToLower())
+        {
+#pragma warning disable SYSLIB0039 // Type or member is obsolete
+            case "tls11":
```

Comment:
Remove? I don't think we will want to do anything with TLS1.1

Reply:
agree, removed